### ISSUE TITLE
@slidy/svelte: direction

### ISSUE THREAD

**Issue by EricRovell** [Valexr/Slidy#65]
# The problem

There should be an option to set a direction as _boolean_ prop `rtl` as some languages have right-to-left direction. Sure, user can traverse the array of items in different direction itself, still, it is a good enhancement to implement. 

## The implementation

There are two options. I think, it should be implemented using new [CSS Logical Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties), as it would use less code instead of just meddling with items order,

**Comment by Valexr** [comment]
Since v3.4.9 @slidy/core automated calculate direction from rendered children flow 🤓
In template just needed css rules, like: `flex-flow: row-reverse`, `direction: rtl` or html attribute on  target node `dir="rtl"`... that's it🤷🏻‍♂️

**Comment by Valexr** [comment]
What do you mean for `direction` ?

**Comment by EricRovell** [comment]
I mean [direction property](https://developer.mozilla.org/en-US/docs/Web/CSS/direction).

**Comment by Valexr** [comment]
> I mean [direction property](https://developer.mozilla.org/en-US/docs/Web/CSS/direction).

Ok, and what...?

**Comment by EricRovell** [comment]
Direction control is implemented, no need for this issue anymore.
